<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>241</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 28pt &quot;Segoe MDL2 Assets&quot;;</string>
      </property>
      <property name="text">
       <string>Menu Principal</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addsongboton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>110</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Song</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>180</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Review Song</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>250</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Download Song</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>320</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Reportes Song</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>390</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Info songs</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>500</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>exit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nombrefield">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>110</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="cantantefield">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>170</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="preciofield">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>220</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>100</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 22pt &quot;Arial Rounded MT Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>160</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 22pt &quot;Arial Rounded MT Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Cantante:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>220</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 22pt &quot;Arial Rounded MT Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Precio:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="generobox">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>270</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>RAP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>POP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ROCK</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DANCE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>REGGAE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ELECTRONICA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RANCHERA</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>260</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 22pt &quot;Arial Rounded MT Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Genero:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addboton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>390</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 22pt &quot;Arial Rounded MT Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 22pt &quot;Arial Rounded MT Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Add Song </string>
      </property>
     </widget>
     <widget class="QLabel" name="etiquetaerror">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>170</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Rockwell&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>310</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 22pt &quot;Arial Rounded MT Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Duracion:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="hora">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>320</y>
        <width>42</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="minutos">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>320</y>
        <width>42</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="segundos">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>320</y>
        <width>42</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>300</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>H</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>300</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>M</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>300</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>S</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>181</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>520</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>exit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="codigoreview">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>141</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="estrellasfield">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>211</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>130</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 22pt &quot;Arial Rounded MT Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Codigo:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>210</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 22pt &quot;Arial Rounded MT Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Estrellas:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="botonreview">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>350</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Arial Rounded MT Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Enviar Review</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 22pt &quot;Arial Rounded MT Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Review Song</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>280</y>
        <width>501</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 300 18pt &quot;Perpetua Titling MT&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>520</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>exit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 22pt &quot;Arial Rounded MT Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Download Song</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Arial Rounded MT Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Codigo de Cancion:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Arial Rounded MT Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Nombre de cliente:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="codigoDescargar">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>120</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="nombredownload">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>180</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="descargarboton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>370</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 22pt &quot;Arial Rounded MT Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Descargar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>250</y>
        <width>0</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>260</y>
        <width>551</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>520</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>exit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 22pt &quot;Arial Rounded MT Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Reportes Song</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>291</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 22pt &quot;Arial Rounded MT Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Nombre de archivo:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nombrearchivo">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="creararchivosboton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>220</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 22pt &quot;Arial Rounded MT Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Crear Archivo</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>110</y>
        <width>381</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>520</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>exit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 22pt &quot;Arial Rounded MT Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Info Songs</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 22pt &quot;Arial Rounded MT Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Codigo :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="codigoInfo">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="verboton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>180</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 22pt &quot;Arial Rounded MT Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Ver</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>100</y>
        <width>421</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="errorlabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>321</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;MS Reference Sans Serif&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
